cpp/include/cudf/io/csv.hpp

```diff
@@ -351,7 +351,7 @@ class csv_reader_options {
    *
    * @param offset Number of bytes of offset.
    */
-  void set_byte_range_offset(size_type offset)
+  void set_byte_range_offset(size_t offset)
```

Comment:
`size_t` should always be qualified with `std`. 
```suggestion
  void set_byte_range_offset(std::size_t offset)
```